templates/Web/_composition/Vue/Page.Vue.AddConstantsFile/.template.config/template.json

```diff
@@ -0,0 +1,30 @@
+﻿{
+    "author": "Microsoft Community",
+    "classifications": [
+      "Universal"
+    ],
+    "name": "Page.Vue.AddConstantsFile",
+  "tags": {
```

Comment:
Should indentation be fixed here?

Reply:
I think the formatting issue is propagate from the same file in React. Maybe we can have a separate PR to fix those formatting together? Or it will be good to fix this one first?

Reply:
Yup this can be fixed. Actually all pages PRs are independent (Once vue base compositions is in my other PRs are independent).

Reply:
In that case yeah we can address the indent issue independently, thanks!